2025/git/01_Git_and_Github_Basics/solution.md

```
@@ -0,0 +1,33 @@
+ git clone https://github.com/surya-edict/ProgressionInDevOps.git
+ git remote add origin https://surya-edict:ghp_Vq1DbFllmXcoqvvAb1OGDKo30TxNF@github.com/surya-edict/ProgressionInDevOps.git
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)